### PR TITLE
app-mobilephone/gammu: bump to 1.41.0

### PR DESCRIPTION
**Pull request by ktrace**
bump version

Closes: https://github.com/gentoo/gentoo/pull/13517
Closes: https://bugs.gentoo.org/696464
Signed-off-by: Victor Kustov <ktrace@yandex.ru>

### CODE REVIEW

**Comment by diogocp** [comment]
> Closes: #13516

No, it doesn't.

File: app-mobilephone/gammu/gammu-1.41.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	bluetooth? ( net-wireless/bluez:= )
+	curl? ( net-misc/curl:= )
+	dbi? ( >=dev-db/libdbi-0.8.3:= )
+	mysql? ( dev-db/mysql-connector-c:= )
+	odbc? ( dev-db/unixODBC )
+	postgres? ( dev-db/postgresql:= )
+	usb? ( virtual/libusb:1= )
+"
+BDEPEND="
+	${DEPEND}
```
Comment (ConiKost) [inline app-mobilephone/gammu/gammu-1.41.0.ebuild]: I don't think, that this is correct, to include all libs into `BDEPEND`..

**Comment by juippis** [comment]
```
99% tests passed, 1 tests failed out of 876

Total Test time (real) = 163.67 sec

The following tests FAILED:
        876 - gammu-detect-run (SEGFAULT)
Errors while running CTest
 * ERROR: app-mobilephone/gammu-1.41.0::testworld failed (test phase):
```
:(
does it pass for you? Need a build.log?

**Comment by ktrace** [comment]
`100% tests passed, 0 tests failed out of 877`
yes, please attach log

**Comment by juippis** [comment]
```
Init:GSM_RegisterAllConnections failed with error UNKNOWNCONNECTIONTYPESTRING[18]: Unknown connection type string. Check config file.
[Gammu            - 1.41.0]
[Connection       - "at"]
[Connection index - 0]
[Model type       - ""]
[Device           - "/NONEXISTING/DEVICE/NODE"]
[Running on       - Linux, kernel 5.3.0-pf5-x86_64 (#1 SMP Wed Oct 30 18:02:31 EET 2019)]
[System error     - open in serial_open, 2, "No such file or directory"]
Init:GSM_TryGetModel failed with error DEVICENOTEXIST[4]: Error opening device, it doesn't exist.
[Gammu            - 1.41.0]
[Connection       - "at-nodtr"]
[Connection index - 0]
[Model type       - ""]
[Device           - "/NONEXISTING/DEVICE/NODE"]
[Running on       - Linux, kernel 5.3.0-pf5-x86_64 (#1 SMP Wed Oct 30 18:02:31 EET 2019)]
[System error     - open in serial_open, 2, "No such file or directory"]
Init:GSM_TryGetModel failed with error DEVICENOTEXIST[4]: Error opening device, it doesn't exist.
[Gammu            - 1.41.0]
[Connection       - "at19200-nopower"]
[Connection index - 0]
[Model type       - ""]
[Device           - "/NONEXISTING/DEVICE/NODE"]
[Running on       - Linux, kernel 5.3.0-pf5-x86_64 (#1 SMP Wed Oct 30 18:02:31 EET 2019)]
[System error     - open in serial_open, 2, "No such file or directory"]
Init:GSM_TryGetModel failed with error DEVICENOTEXIST[4]: Error opening device, it doesn't exist.
```

probably related, it's good. Thanks!

**Pull request closed without merging**